eclair-core/src/main/scala/fr/acinq/eclair/router/Announcements.scala

```diff
@@ -76,15 +76,16 @@ object Announcements {
       case address@(_: Tor2) => (3, address)
       case address@(_: Tor3) => (4, address)
     }.sortBy(_._1).map(_._2)
-    val witness = nodeAnnouncementWitnessEncode(timestamp, nodeSecret.publicKey, color, alias, features, sortedAddresses, TlvStream.empty)
+    val nodeAnnouncementFeatures = features.nodeAnnouncementFeatures()
```

Comment:
Yes agreed, we should have done this earlier! I think doing it now shouldn't harm anyone, and we should do it asap because in the future it may become more harmful.